src/TrackerTable.tsx

```diff
@@ -2,6 +2,14 @@ import React from 'react';
 import { calculateTotalTimeSpent, calculateAverageTimeSpent } from './utils';
 import { Item } from './types';
 import { Discipline } from './types/constants';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+import { TableFooter } from '@mui/material';
```

Comment:
why do you use different type of imports? import X from Y; or import {qwe} from Y ???